cluster/gce/gci/configure-helper.sh

```diff
@@ -3115,7 +3115,7 @@ spec:
     - /bin/sh
     args:
     - -c
-    - test -e /scrub && rm -rf /scrub/..?* /scrub/.[!.]* /scrub/* && test -z $(ls -A /scrub) || exit 1
+    - test -e /scrub && find /scrub -mindepth 1 -delete && test -z $(ls -A /scrub) || exit 1
```

Comment:
Would it make sense to use the -ignore_readdir_race option with the 'find -delete' ?